src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -153,6 +324,15 @@ def __init__(self, help_ctx, delimiters, parser):
         help_param.group_name = 'Global Arguments'
 
     def _load_from_data(self, data):
+
+        def _params_equal(data, param):
+            for name in param.name_source:
+                if data.get("name") == name.lstrip("-"):
+                    return True
+            return False
```

Comment:
This is a breaking change from the previous behavior- if `data.get('name')` was "-f --force", this would return false

Reply:
So, I just tested this scenario and it works. If the data was obtained from _help.py, then HelpFile.load() is called which in turn calls CommandHelpFile._load_from_data() which ensures the previous behavior. 

The change to the previous behaviour is that _help.py will now also accept `f` or `force`